### ISSUE TITLE
Develop generic needs assessment

### ISSUE THREAD

**Issue by tareko** [GliaX/meta#1]
Currently, here is the needs assessment draft:

```
- [ ] **Gut check**: Does it make sense to make this device
- [x] **Other people's gut check**: Talk to at least 2 other clinical physicians to ask if these devices are actually needed
- [ ] **Target people's gut check**: Talk to at least 2 'on-the-ground' physicians in high-needs areas to ask if these devices are actually needed
- [ ] **Price check**: How much does the device currently cost?
- [ ] **Price reduction**: How much do we anticipate we can drop the price?
- [ ] **Safety check**: Can we make this product safely for the builders and for the users?
```

Linked to https://github.com/FreeMedicalHardware/pulseox/issues/3

Here is the task list for completion of this issue:
- [x] @tareko to develop first draft
- [ ] @amashari to review
- [ ] Send to external reviewers for comment
  - [ ] Who?
- [ ] Send to group for final comment
- [ ] Commit to wiki

**Comment by amymiller** [comment]
**Price check**: How much does the device currently cost? 
The leading product currently available in the stethoscope market is the Littmann line
made by 3M, which currently sells for between 75$-500$US per unit.
There are currently hundreds of Pulse oximeter available for sale with prices ranging
from 30$-350$US
There are numerous ECG's available for sale with prices ranging from 130$-1250$US

**Comment by tareko** [comment]
@amymiller The idea here is to develop a template for use when we come up with other devices. And so, device-specific information doesn't belong in this ticket. Ostensibly, each device should have its information there.

However, it might be an idea to amalgamate all of the needs assessments within the Meta repository somewhere. Thoughts?

**Comment by amymiller** [comment]
@tareko Makes sense to me however I am not sure how to move forward with this issue. I have it as an issue to discuss when we talk next.
